invisible_cities/cities/penthesilea_test.py

```diff
@@ -359,3 +359,30 @@ def test_penthesilea_empty_input_file(config_tmpdir, ICDATADIR):
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", category=UserWarning)
         penthesilea(**conf)
+
+
+
+def test_penthesilea_global_xyreco_bias(config_tmpdir, ICDATADIR):
+    # Make sure the global_reco_params are used for the columns Xpeak/Ypeak
+    # This is done by running over an event that has many spurious sipm hits
+    # and only one sipm with signal over the threshold. If the threshold
+    # is used, the reconstructed position will match the sipm position.
+    # If the threshold is not used the position will be biased.
+
+    PATH_IN  = os.path.join(ICDATADIR    , 'fake_pmap_nothreshold_barycenter_bias.h5')
+    PATH_OUT = os.path.join(config_tmpdir, 'fake_hdst_nothreshold_barycenter_bias.h5')
+
+    conf = configure('dummy invisible_cities/config/penthesilea.conf'.split())
+    conf.update(dict(files_in    = PATH_IN ,
```

Comment:
Maybe you want to change the comma alignment following https://github.com/next-exp/IC/pull/776#discussion_r584109590

Reply:
Is it agreed to change to this alignment style?

Reply:
Given that we have

+ In favour:
   1. my original suggestion
   2.  @gondiaz' heart emoji in response to said suggestion
   3.  @Aretno's suggestion that you use the style
+ against:
   ∅

that is to say, the ratio of (implicit or explicit) favourable to unfavourable opinions is ∞, I'm tempted (in the absence of vehement opposition in the next 24 hours) to conclude that we've 'agreed' to change to this style. Do you have an opinion on it?

This is a choice which has no profound and irreversible consequences. No point in waiting on a decision. Just Do It! (Unless you hate it.)

Reply:
> Do you have an opinion on it?

I'm in 100% favor of this alignment style. I was just wondering whether it was official or not. I am guessing we will not make the effort of realigning existing code, but rather request new code to follow this rule, right?

Reply:
> I'm in 100% favor of this alignment style.

Quadruple infinity. I think that settles it.

> I was just wondering whether it was official or not.

AFAIAC, it is now.

> I am guessing we will not make the effort of realigning existing code, but rather request new code to follow this rule, right?

Exactly.

We can also adapt existing code when we are changing it for some other purpose.